code/modules/reagents/chemistry/recipes/food.dm

```diff
@@ -102,6 +102,22 @@
 	for(var/i = 1, i <= created_volume, i++)
 		new /obj/item/reagent_containers/food/snacks/dough(location)
 
+///Cookies by Ume
+
+/datum/chemical_reaction/cookiedough
+	name = "Dough"
+	id = "dough"
+	result = null
+	required_reagents = list("milk" = 10, "flour" = 10, "sugar" = 5)
+	result_amount = 1
+	mix_message = "The ingredients form a dough. It smells sweet and yummy."
+
+/datum/chemical_reaction/cookiedough/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
```

Comment:
Please use the for(var/i in 1 to created_volume) - its faster and in our style guide. It is also easier to understand

Reply:
I used it as it was like this for the normal dough, should I change the normal dough too?

Reply:
(Sorry forgot to switch accounts before replying, I'm working on two separated projects with the same code, so I had to fork something twice with different accounts)